### PR TITLE
Fix/create second wallet

### PR DESCRIPTION
**Pull request by antonijzelinskij**
#8694e6tqy
#8694fkf4d

### CODE REVIEW

**Comment by stepanLav** [comment]
Task linked: [CU-8694e6tqy Android. Create a new wallet from Settings](https://app.clickup.com/t/8694e6tqy)
Task linked: [CU-8694fkf4d Android. Bottom sheet to remind user about Cloud backup each time user changed backup (Create Wallet, Update Wallet, Remove Wallet)](https://app.clickup.com/t/8694fkf4d)

File: feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/account/details/WalletDetailsInteractor.kt [hunk 1]
```diff
@@ -32,32 +34,33 @@ class WalletDetailsInteractor(
         accountRepository.updateMetaAccountName(metaId, newName)
     }
 
-    suspend fun getChainProjections(
-        metaAccount: MetaAccount,
+    fun chainProjectionsFlow(
```
Comment (antonijzelinskij) [inline feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/account/details/WalletDetailsInteractor.kt:37]: Change it to flow to support realtime meta account changes

File: feature-onboarding-impl/src/main/java/io/novafoundation/nova/feature_onboarding_impl/domain/OnboardingInteractorImpl.kt [hunk 2]
```diff
@@ -12,7 +14,9 @@ class OnboardingInteractorImpl(
     }
 
     override suspend fun isCloudBackupAvailableForImport(): Boolean {
-        return !cloudBackupService.session.isSyncWithCloudEnabled()
+        val syncWithCloudNotEnabled = !cloudBackupService.session.isSyncWithCloudEnabled()
+        val withoutAddedAccounts = !accountRepository.hasActiveMetaAccounts()
```
Comment (valentunn) [inline feature-onboarding-impl/src/main/java/io/novafoundation/nova/feature_onboarding_impl/domain/OnboardingInteractorImpl.kt:18]: Why do we limit it like that? It should still be possible to import from backup even when we already have accounts added. We specifically designed a sync strategy for it to merge local and cloud accoutns in a way suitable for import

File: feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/presentation/startCreateWallet/StartCreateWalletPayload.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+class StartCreateWalletPayload(
+    val flowType: FlowType
+) : Parcelable {
+
+    enum class FlowType {
+        FIRST_WALLET,
+        SECOND_WALLET
```
[the remark this answers is not in the document]
Comment (antonijzelinskij) [reply feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/presentation/startCreateWallet/StartCreateWalletPayload.kt:13]: I think it's equivalent, but my approach is more explicit

File: feature-onboarding-impl/src/main/java/io/novafoundation/nova/feature_onboarding_impl/domain/OnboardingInteractorImpl.kt (hunk 2 above)
Comment (antonijzelinskij) [reply feature-onboarding-impl/src/main/java/io/novafoundation/nova/feature_onboarding_impl/domain/OnboardingInteractorImpl.kt:18]: I followed our design layouts. Apparently, if we have added wallets, we can only enable backup from the settings

Comment (valentunn) [reply feature-onboarding-impl/src/main/java/io/novafoundation/nova/feature_onboarding_impl/domain/OnboardingInteractorImpl.kt:18]: From personal messages: we return such a possibility as it was confirmed on the planning

**Review by valentunn — APPROVED** [review]

**Pull request merged**